active-rfcs/0000-finalize-head-body.md

```diff
@@ -0,0 +1,80 @@
+- Start Date: 11/30/2021
+- Reference Issues: https://github.com/withastro/rfcs/discussions/15
+- Implementation PR:
+
+# Summary
+
+Finalize behavior around `<head>`, `<body>`, `<html>`, and `<!DOCTYPE html>` elements in Astro component templates.
+
+# Motivation
+
+in Astro v0.21, we have had user reports around `<head>` not acting as expected, breaking some projects from v0.20:
+- https://github.com/withastro/astro/issues/2046
+- https://github.com/withastro/astro/issues/2022
+
+Some of these issues stem from undocumented or undefined behaviors in v0.20: was it allowed for `<head>` be conditionally added in a nested expression? 
+
+Other issues stem from our attempt to provide implicit `<head>` and `<body>` support in pages and layouts. There was a quick workaround added for v0.21 where `src/pages` and `src/layouts` folder locations must be defined in config so that our parser knows how to parse them as full HTML documents and not smaller HTML fragments. This unblocked users but at the expense of breaking two larger design goals of the project:
+  - Added `src/layouts` as a new special folder location, when only `src/pages` was intented to be "special".
+  - Caused Astrp to create different output for a component based on the file path of that component.
```

Comment:
typo `Astrp`

Reply:
Quickly registers the astrp package, astrp dot org, begins selling astrp merch, etc.

Reply:
dang, https://github.com/astrp is already taken! :D